packages/core/functions/src/services/remote-function-execution-service.ts

```diff
@@ -14,10 +14,15 @@ import { getInvocationUrl } from '../url';
 export class RemoteFunctionExecutionService extends Context.Tag('@dxos/functions/RemoteFunctionExecutionService')<
   RemoteFunctionExecutionService,
   {
-    callFunction(deployedFunctionId: string, input: any, spaceId?: SpaceId): Promise<any>;
+    callFunction(deployedFunctionId: string, input: any): Promise<any>;
```

Comment:
add a good jsdoc explaining what `deployedFunctionId` is. is it the key? is it the id of the function object. is the guid?